src/main/java/com/manardenza/exceptions/DbFileReadException.java

```diff
@@ -0,0 +1,8 @@
+package com.manardenza.exceptions;
+
+public class DbFileReadException extends RuntimeException {
+
+    public DbFileReadException(String message, Throwable cause) {
```

Comment:
1. Why do you need two separate exceptions for similar operation?
2. Why do you need cause here?